ccpp/data/GFS_typedefs.meta

```diff
@@ -28,14 +28,14 @@
   type = real
   kind = kind_phys
 [prsik]
-  standard_name = dimensionless_exner_function_at_model_interfaces
+  standard_name = dimensionless_exner_function_at_interface
   long_name = dimensionless Exner function at model layer interfaces
   units = none
-  dimensions = (horizontal_loop_extent,vertical_dimension_plus_one)
+  dimensions = (horizontal_loop_extent,vertical_interface_dimension)
   type = real
   kind = kind_phys
 [prsik(:,1)]
-  standard_name = dimensionless_exner_function_at_lowest_model_interface
+  standard_name = surface_dimensionless_exner_function
   long_name = dimensionless Exner function at lowest model interface
```

Comment:
Exner function by definition is dimensionless.  So why add redundancy in the name?

Reply:
CF has "dimensionless_exner_function" so we are more-or-less forced to adopt this if our standard names are to be a superset of theirs. This is a good question that we plan to ask the CF standards committee, however, and we had the same question. In fact, we've collected several questions for them regarding their names, but have not tried to get anything changed yet.